src/manifest.json

```diff
@@ -35,7 +35,10 @@
   },
   "permissions": [
     "activeTab",
-    "storage"
+    "storage",
+    "contextMenus",
+    "http://*/",
+    "https://*/"
```

Comment:
Are these necessary? I think since we have
```
"content_scripts": [
  {
    "matches": ["<all_urls>"],
```

above this should already be assumed.